.github/workflows/continuous-delivery.yml

```diff
@@ -82,61 +103,26 @@ jobs:
 
       ## Deploy
       - name: Deploy packages
-        run: mvn -B deploy -DskipTests -Dmaven.install.skip=true
+        run: mvn ${{ env.MAVEN_CLI_OPTS }} deploy -Dmaven.install.skip=true
 
+      ## Create Release
       - name: Create Release
         id: create_release
-        uses: actions/create-release@v1
-        env:
-          #don't use the GITHUB_TOKEN https://github:community/t/actions-not-triggered-on-release/17944/12:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        uses: softprops/action-gh-release@v1
         with:
+          name: Release ${{ env.NEW_VERSION }}
```

Comment:
ich würde `Release` weglassen. Aber wenn ihr das im core team so haben wollt und das auch in den anderen Produkten schon ist, geht das natürlcih.

Reply:
Ich hab das von den anderen Repos so übernommen.

Reply:
Ich kann mit beiden Varianten leben.
Wenn ich es wegmachen soll, gib einfach Bescheid